dev_guide/deployments.adoc

```diff
@@ -352,3 +352,16 @@ Rollbacks revert an application back to a previous deployment and can be
 performed using the REST API or the CLI. See the
 link:../cli_reference/basic_cli_operations.html#deployment-operations[CLI
 Reference] for more details.
+
+== Fine-Grained Control
```

Comment:
@tnguyen-rh Thoughts on this title versus something more straight-forward like "Resizing"?